try: 
    print(10/0)
except Exception as e:
    print("error is: ", e)

     